windows/CodePushUtils.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.System.Profile;
+
+namespace CodePush.ReactNative
+{
+    class CodePushUtils
+    {
+        public static readonly string REACT_NATIVE_LOG_CATEGORY = "ReactNative";
+
+        public async static Task<JObject> GetJObjectFromFile(StorageFile file)
+        {
+            string jsonString = await FileIO.ReadTextAsync(file);
+            if (jsonString.Length == 0)
+            {
+                return new JObject();
+            }
+
+            return JObject.Parse(jsonString);
```

Comment:
Because this is an internal method, I think it should be the caller's responsibility to give a file that has valid JSON written in it, and if not, a JSON parsing exception should be thrown. `try... catch`-ing it and returning `null` will make the error harder to diagnose.